### DISCUSSION TITLE
how to fix it plz help me !

### DISCUSSION THREAD

**Discussion by brownspy1** [deta/cloud-docs#330] [Help]
![image](https://user-images.githubusercontent.com/89521172/180064622-46621463-b11b-4f7b-974b-903ef92771e3.png)

**Comment by Satyab21** [comment]
maybe.. but try the following steps (~6min process) below: 

there are 2 `.deta` folders.
(1. ~1min) delete .deta where you `installed` it with the `iwr... ` command, it might be installed in dir like `C drive/user/abc/.deta` [this is NOT the .deta folder you get from `deta new --python --name xyz`]. Delete it, as simple as going to `C:` and press Delete using keyboard.  
(2. ~1min) After deleting the first iwr based .deta, the now you will need to delete the second `.deta` folder that was created from the `deta new --python --name xyz` command. This .deta folder might be in your project folder or where ever you are using it.  hit Delete on keyboard.

(3. ~1min) reinstall the .deta cli using the `iwr https://get.deta.dev/cli.ps1 -useb | iex` command for windows or other [os](https://docs.deta.sh/docs/cli/install). 
(4. ~1min) Once you complete the iwr cli installation, test it out on the same terminal using `deta --help` to see if it is recognized by your os. if it did recognize (as it should for the most part).. then now move on to next step.5 && keep this step.4 terminal open tho.. dont close it off.
(5. ~2min) open a new terminal, while keeping the previous one open,- use this terminal and `cd` to your project folder now- like in desktop or D: drive or E:... if your project uses a virtual env activate it using this step.5 terminal.. once activated use this same step.5 terminal to create a new micro `deta new  --python --name project/new-micro-name`. 

now try running `deta deploy` or whatever you need to do

TLDR (~ 1min): 
(1) **delete** cli installed .deta 
(2) **delete** project micro .deta 
(3 ) **install** cli .deta
(4) **create** project .deta

**Comment by BenitzCoding** [comment]
Try re-opening the terminal after installing Deta.
